packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -190,6 +196,8 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 					quickReplies: msg.quickReplies,
 					read,
 				});
+			} else if (msg.type === 'intermediate-step') {
+				assistantThinkingMessage.value = locale.baseText(msg.step);
```

Comment:
what if key does not exist in a version? Should not the backend return the value itself instead of localization key? 

if we were to add localization in the future, the backend would return it localized

Reply:
Yeah, my main concern here is the localization but I agree that should be handled by the service. Will update.